src/game/SpellEffects.cpp

```diff
@@ -446,6 +446,27 @@ void Spell::EffectDummy(SpellEffectIndex eff_idx)
                     m_caster->CastSpell(m_caster, spell_id, true, nullptr);
                     return;
                 }
+                case 8344:                                  // Gnomish Universal Remote (ItemID: 7506)
+                {
+                    Unit* channelTarget = m_originalCaster->GetMap()->GetUnit(m_originalCaster->GetChannelObjectGuid());
```

Comment:
You need check m_originalCaster != null before use. In some cases GUID can't be resolved to pointer in time effect apply